site/docusaurus.config.js

```diff
@@ -14,172 +14,181 @@ const theButton = require('./src/rehype/thebutton');
+              position: 'right',
+            },
+            {
+              href: 'https://discord.com/invite/machGGczea',
+              position: 'right',
+              className: 'header-discord-link',
+              'aria-label': 'Community',
+            },
+            {
+              href: 'https://github.com/iasql/iasql-engine',
+              position: 'right',
+              className: 'header-github-link',
+              'aria-label': 'GitHub repository',
+            },
+            {
+              type: 'search',
+            },
+          ],
         },
-        blog: {
-          showReadingTime: true,
-          routeBasePath: '/blog',
-          remarkPlugins: [theButton]
+        footer: {
+          style: 'light',
+          links: [
+            {
+              title: 'Product',
+              items: [
+                {
+                  label: 'Docs',
+                  href: 'docs',
+                  target: '_self',
+                },
+                {
+                  label: 'Blog',
+                  href: 'blog',
+                  target: '_self',
+                },
+                {
+                  label: 'Dashboard',
+                  href: 'https://app.iasql.com',
+                  target: '_self',
+                },
+                {
+                  label: 'GitHub',
+                  href: 'https://github.com/iasql/iasql-engine',
+                  target: '_self',
+                },
+              ],
+            },
+            {
+              title: 'Community',
+              items: [
+                {
+                  label: 'Discord',
+                  href: 'https://discord.com/invite/yxNBQugGbH',
+                },
+                {
+                  label: 'Twitter',
+                  href: 'https://twitter.com/iasql',
+                },
+              ],
+            },
+          ],
         },
-        theme: {
-          customCss: require.resolve('./src/css/custom.css'),
+        prism: {
+          additionalLanguages: [],
+          theme: lightCodeTheme,
+          darkTheme: darkCodeTheme,
         },
-        sitemap: {
-          changefreq: 'weekly',
-          priority: 0.5,
-          ignorePatterns: ['/tags/**'],
-          filename: 'sitemap.xml',
+        algolia: {
+          // The application ID provided by Algolia
+          appId: '5D0XPJ20G6',
+          // Public API key: it is safe to commit it
+          apiKey: '3975e3fd292172a78a2092aecca9a2b5',
+          indexName: 'docs-iasql',
+          contextualSearch: true,
         },
       }),
-    ],
-  ],
 
-  plugins: [
-    [
-      "posthog-docusaurus",
+    scripts: [
       {
-        apiKey: config.phKey,
-        appUrl: "https://app.posthog.com", // optional
-        enableInDevelopment: true, // optional
-        // other options are passed to posthog-js init as is
+        src: 'https://cdnjs.cloudflare.com/ajax/libs/js-cookie/3.0.1/js.cookie.js',
```

Comment:
i need to have a library to set cookies... so i can just call Cookies.set in one-liner from the button. It also simplifies the expiration and domain for the cookie, rather than doing it with plain javascript

Reply:
I prefer to avoid using/depending on 3rd parties when possible. In this case, I think it could still be done in a clean way using the `document.cookie` API https://developer.mozilla.org/en-US/docs/Web/API/Document/cookie

Reply:
I just added the files as static dependencies. I think that the document.cookie generation is too complex to just include in one line (especially for managing the expiration in +1 day). So either i include that js.cookie statically or i write my own function... but i'd prefer to manage cookies via js function.
We also need to include lz-string to compress, so i also added as an static dep